apps/fhir/bluebutton/v3/urls.py

```diff
@@ -0,0 +1,54 @@
+from django.urls import re_path
+from django.contrib import admin
+
+from apps.fhir.bluebutton.views.read import (
+    ReadViewCoverage,
+    ReadViewExplanationOfBenefit,
+    ReadViewPatient,
+)
+from apps.fhir.bluebutton.views.search import (
+    SearchViewCoverage,
+    SearchViewExplanationOfBenefit,
+    SearchViewPatient,
+)
+
+admin.autodiscover()
+
+urlpatterns = [
+    # Patient ReadView
+    re_path(
+        r"Patient/(?P<resource_id>[^/]+)",
+        ReadViewPatient.as_view(version=2),
+        name="bb_oauth_fhir_patient_read_or_update_or_delete_v2",
```

Comment:
Should we go ahead and use `version=3` and `bb_oath_fhir_..._v3` for these? Just to start separating things out to make it easier to swap over to BFD v3 when that time comes?